### DISCUSSION TITLE
Is there anyone here that working with Swift by using Neovim?

### DISCUSSION THREAD

**Discussion by bengidev** [LazyVim/LazyVim#2372] [General]
I've been struggling to setup Neovim configuration to be able to work with Swift. If anyone in here work with Swift, please help me to setup my Neovim config for Swift. 

This is my repo for my current NeoVim config:
https://github.com/bengidev/Bengi_LazyVim_Configuration

Thanks for your help 🍻

**Comment by karimlevallois** [comment]
https://github.com/wojciech-kulik/ios-dev-starter-nvim
